pkg/conf/clusters.go

```diff
@@ -74,10 +75,20 @@ func ReadClustersConfig(r io.Reader) (Clusters, error) {
 		if len(cluster.Hosts) == 0 && cluster.Type != BlackholeCluster {
 			return cls, fmt.Errorf("cluster with index %d does not have any hosts", idx)
 		}
-		for hostIdx, host := range cluster.Hosts {
+
+		inxs := map[int]bool{}
+		for _, host := range cluster.Hosts {
 			if host.Name == "" {
-				return cls, fmt.Errorf("host %d in cluster with index %d does not have a name", hostIdx, idx)
+				return cls, fmt.Errorf("host %d in cluster does not have a name", idx)
```

Comment:
I assume this log can be a bit misleading. e.g. with config:
```[[cluster]]
		name = "aaa"
		type = "jump"
			[[cluster.hosts]]
			name = "host1"
			index = 0
			port = 123
			[[cluster.hosts]]
			name = ""
			index = 1
			port = 456
```
			

We get `host 0 in cluster does not have a name`.